csrc/velox/functions/functions.h

```diff
@@ -226,6 +227,15 @@ inline void registerTorchArrowFunctions() {
       velox::Array<int64_t>,
       velox::Array<int64_t>>({"bucketize"});
 
+  // TorchText
+  // bpe_encode
+    velox::registerFunction<
+      bpe_tokenize,
+      velox::ArrayWriterT<int64_t>,
+      std::shared_ptr<GPT2BPEEncoder>,
+      velox::Varchar>({"bpe_tokenize"});
```

Comment:
nit: you can name the struct name as `BpeTokenizeFunction` since it will be aliased to `bpe_tokenize` anyway here.